mifosng-android/src/main/java/com/mifos/mifosxdroid/online/RunReportsActivity.java

```diff
@@ -27,35 +30,45 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayShowTitleEnabled(false);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
-        Spinner spinner = new Spinner (getSupportActionBar().getThemedContext());
+        Spinner spinner = new Spinner(getSupportActionBar().getThemedContext());
 
         ArrayAdapter<CharSequence> adapter = ArrayAdapter.createFromResource(this,
                 R.array.array_runreport, R.layout.simple_spinner_item);
         adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
         spinner.setAdapter(adapter);
         spinner.setOnItemSelectedListener(this);
-
         toolbar.addView(spinner);
+        intent = new Intent("report");
```

Comment:
Don't use any Action like this. From the [documentation](https://developer.android.com/reference/android/content/Intent) : 

> Note that the action must be in a namespace because Intents are used globally in the system -- for example the system VIEW action is android.intent.action.VIEW; an application's custom action would be something like com.google.app.myapp.CUSTOM_ACTION